recipes-kernel/kernel-tests/kernel-tests-files/kernel_dmesg_diff.py

```diff
@@ -59,27 +59,31 @@ def __init__(self):
         self.major_minor = kernel_version.group(2)
         self.type = 'next' if kernel_version.group(3) else 'current'
 
+class OsVersion:
+    def __init__(self):
+        with open('/etc/os-release', 'rb') as file, mmap.mmap(file.fileno(), 0, access=mmap.ACCESS_READ) as mfile:
```

Comment:
(MINOR) Could you [add an RDEPENDS](https://github.com/ni/meta-nilrt/blob/nilrt/master/kirkstone/recipes-kernel/kernel-tests/kernel-tests.bb#L14) on `os-release`?

Reply:
Done.